src/storage/examples/src/client/configure_retries.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_configure_retries]
+use google_cloud_gax::exponential_backoff::ExponentialBackoffBuilder;
+use google_cloud_gax::retry_policy::RetryPolicyExt;
+use google_cloud_storage::client::{Storage, StorageControl};
+use google_cloud_storage::retry_policy::RetryableErrors;
+use std::time::Duration;
+
+pub async fn sample(bucket_id: &str) -> anyhow::Result<()> {
+    // Retries all operations for up to 5 minutes, including any backoff time.
+    let retry_policy = RetryableErrors.with_time_limit(Duration::from_secs(60 * 5));
+    // On error, it backs off for a random delay between [1, 3] seconds, then [3,
```

Comment:
This is not true I believe:

https://github.com/googleapis/google-cloud-rust/blob/0f870d1dce4af597602b961d500d100123742496/src/gax/src/exponential_backoff.rs#L210

The detail is first in the range $[0, 1]$, then in $[0, 3]$, then in $[0, 9]$, etc.

Reply:
http://cloud/storage/docs/samples/storage-configure-retries Ah I had referenced the C++ sample, which I guess works very differently (or is also wrong I will go check)! Will update the comment, thanks for catching

Reply:
C++ example looks correct for that implementation.

Fixed this sample to match our implementation.

Reply:
https://github.com/googleapis/google-cloud-cpp/issues/8755 probably has the background for C++.